dc/core/resources/BorderRectangle.qml

```diff
@@ -39,7 +39,7 @@ Item {
         width: Style.borderWidth
         height: Style.borderWidth
         radius: Style.borderWidth
-        color: Style.inactiveBorderColor
+        color: "#00000000"
```

Comment:
It should described what it is used for, not the value. Style.borderColor? Or why not changing the value of inactiveBorderColor (which is unused now after your change?)?